modules/ossm-release-2-6-2.adoc

```diff
@@ -0,0 +1,46 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-6-2_{context}"]
+= {SMProductName} version 2.6.2
+
+This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.2, and includes the following ServiceMeshControlPlane resource version updates: 2.6.2, 2.5.5 and 2.4.11.
+
+This release addresses Common Vulnerabilities and Exposures (CVEs); contains bug fixes; and is supported on {product-title} 4.14 and later.
```

Comment:
These were [semicolons in 2.6.1](https://82442--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes#ossm-release-2-6-1_ossm-release-notes) because the list included a list. Should the 2.6.2 intro match the 2.6.1 intro more directly? If not, replace these with commas:
```suggestion
This release addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.14 and later.
```

Personally, I would lean towards a consistent format because the Service Mesh RNs are inherently a bit confusing by virtue of the product release method, and consistent presentation can help with that :sweat_smile: Might be worth creating a template for this at some point :thinking: